src/main/java/frc/robot/hardware/LedMatrixImages.java

```diff
@@ -97,6 +97,6 @@ public class LedMatrixImages {
 {"#000000", "#000000", "#000000", "#000000", "#000000", "#ffff00", "#ffff00", "#ffff00", "#ffff00", "#ffff00", "#ffff00", "#000000", "#000000", "#000000", "#000000", "#000000"}, 
 {"#000000", "#000000", "#000000", "#000000", "#000000", "#ffff00", "#ffff00", "#ffff00", "#ffff00", "#ffff00", "#ffff00", "#000000", "#000000", "#000000", "#000000", "#000000"}, 
 {"#000000", "#000000", "#000000", "#000000", "#000000", "#ffff00", "#ffff00", "#ffff00", "#ffff00", "#ffff00", "#ffff00", "#000000", "#000000", "#000000", "#000000", "#000000"}, 
-{"#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000"}
+{"#000000", "#000000", "#000000", /*In solidarity may the real CAN shady please stand up*/ "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000", "#000000"}
```

Comment:
Well you've got it into the Git commit history. But it ain't staying for the merge...

Reply:
Removed it